bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ResourceComponent.java

```diff
@@ -31,10 +31,6 @@
  */
 public class ResourceComponent extends UIOutput {
 
-	public ResourceComponent() {
```

Comment:
@ngriffin7a, I could not reproduce the error that cause me to add this. I don't think it's necessary right now. But we should check it if we are getting weird exception when Primefaces or Icefaces try to restore state.